resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -34,10 +44,68 @@ var versionCmd = &cobra.Command{
 	Short: "Print the version number of gc",
 	Long:  `All software has versions. This is gc version's`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("{{packageVersion}}")
+		fmt.Println("Current version: {{packageVersion}}")
+		checkForNewVersion()
 	},
 }
 
+func checkForNewVersion() {
+	var url string
+	if strings.Contains("{{basePath}}", "api.mypurecloud.com") {
+		url = externalVersionURL
+	} else {
+		url = internalVersionURL
+	}
+
+	err, latestVersion := retrieveLatestVersion(url)
+	if err != nil {
+		fmt.Println("An error occurred while retrieving the latest version.")
+		return
+	}
+
+	if versionsAreEqual("17.0.0", latestVersion) {
+		fmt.Println("You're all up to date.")
+	} else {
+		fmt.Printf("A new version of the CLI is available: %v\n", latestVersion)
+	}
+}
+
+func retrieveLatestVersion(url string) (error, string) {
+	client := http.Client{
+		Timeout: time.Second * 2,
+	}
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		return err, ""
+	}
+
+	res, getErr := client.Do(req)
+	if getErr != nil {
+		return getErr, ""
+	}
+	if res.Body != nil {
+		defer res.Body.Close()
+	}
+	body, readErr := ioutil.ReadAll(res.Body)
+	if readErr != nil {
+		return readErr, ""
+	}
+
+	latestVersion := LatestVersion{}
+	jsonErr := json.Unmarshal(body, &latestVersion)
+	if jsonErr != nil {
+		return jsonErr, ""
+	}
+
+	return nil, latestVersion.Version
+}
+
+func versionsAreEqual(current string, latest string) bool {
+	v1, _ := version.NewVersion(current)
+	v2, _ := version.NewVersion(latest)
+	return v1.Equal(v2)
```

Comment:
GreaterThanOrEqual would be safest here in case for some reason the latest version file isn't updated with the latest version or there's a caching issue.

Also it would be a good idea to write a unit test for this function.